backend/src/test/java/io/debezium/configserver/util/Infrastructure.java

```diff
@@ -65,10 +65,19 @@ public enum DATABASE {
                     .withEnv("MYSQL_ROOT_PASSWORD", "debezium")
                     .withNetworkAliases("mysql");
 
-    private static final MongoDBContainer MONGODB_CONTAINER =
-            new MongoDbContainer(DockerImageName.parse("mongo:3.6"))
-                    .withNetwork(NETWORK)
-                    .withNetworkAliases("mongodb");
+    private static final MongoDbContainer MONGODB_CONTAINER;
+    static {
+        MONGODB_CONTAINER = MongoDbContainer.node()
+                .name("localhost")
+                .replicaSet("rs0")
+                .imageName(DockerImageName.parse("mongo:5.0"))
+                .network(NETWORK)
+                .build();
+        MONGODB_CONTAINER
+                .withNetworkAliases("mongodb")
+                .withCopyFileToContainer(MountableFile.forClasspathResource("initialize-mongo-single.js"), "/docker-entrypoint-initdb.d/")
+                .initReplicaSet(false, MONGODB_CONTAINER.getNamedAddress());
```

Comment:
```suggestion
                .build()
                .withNetworkAliases("mongodb")
                .withCopyFileToContainer(MountableFile.forClasspathResource("initialize-mongo-single.js"), "/docker-entrypoint-initdb.d/");
                MONGODB_CONTAINER.initReplicaSet(false, MONGODB_CONTAINER.getNamedAddress());
```

Reply:
And shouldn't the `MONGODB_CONTAINER.initReplicaSet(false, MONGODB_CONTAINER.getNamedAddress());` command go to the `io.debezium.configserver.util.MongoDbInfrastructureTestResourceLifecycleManager#start` method? A replica set can only be initialized for a started container, right?

Reply:
This won't work because these methods require a started container, I've delayed this until after the container has started.